app/javascript/components/LoadingSpinner.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import 'stylesheets/loading-spinner';
+
+const LoadingSpinner = ({ className }) => (
```

Comment:
I know it technically isn't a "spinner," so... I'm open to better name suggestions?

Reply:
`LoadingStatus`? haha idk I'm fine with it either way

Reply:
I'll leave it for now and if someone wants to fight me they totally can